src/main/java/com/example/fooddelivery/repository/ProductRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.example.fooddelivery.repository;
+
+import com.example.fooddelivery.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
+
+    List<Product> findAllByRestaurant_Id(Long id);
```

Comment:
why you used '_' in method signature? can't we use only camel case?

Reply:
This is how the autocomplete suggests to me.  I will test if its work without underscore.